data/json/furniture_and_terrain/furniture-storage.json

```diff
@@ -882,7 +882,7 @@
     "type": "furniture",
     "id": "f_dumpster",
     "name": "dumpster",
-    "description": "Stores trash.  Doesn't get picked up anymore.  Note the smell.",
+    "description": "A large metal dumpster that will likely not be getting picked up by the city's waste management any time soon. Despite the unpleasant nature of climbing inside, it could make for a viable hiding spot.",
```

Comment:
```suggestion
    "description": "A large metal dumpster that will likely not be getting picked up by the city's waste management any time soon.  Despite the unpleasant nature of climbing inside, it could make for a viable hiding spot.",
```